admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -12,6 +12,8 @@
 <header>
   <nav class="main-header">
 
+    <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
```

Comment:
I pick it from ui-kit http://build.prestashop.com/prestashop-ui-kit/section-11.html